src/Common/Enum/Enum.php

```diff
@@ -35,13 +35,26 @@ public function __construct($value);
      */
     public static function __callStatic(string $enum, array $args = []): Enum;
 
+    /**
+     * Return the enum's valid values
+     *
+     * @return mixed
+     */
+    public static function getValues(): array;
```

Comment:
Method `getValues()` is too similar to `getValue()`, this can create confusion.
Any suggestion for a different name?

Reply:
Let me think about it 😄

Reply:
+1 It's better than current name